airmozilla/main/fields.py

```diff
@@ -1,7 +1,7 @@
 from django.db import models
 from django.forms.fields import Field
 from django.forms.util import ValidationError
-from django.utils import simplejson as json
+import json
```

Comment:
Unfortunately, it seems a race between you and @aarshvig on the same patch. 
Please see the comment I made [on her PR](https://github.com/mozilla/airmozilla/pull/224/files#r26050276) about the order of the imports.